stations.csv

```diff
@@ -43167,108 +43167,108 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
+47319;Genève Intercontinental;geneve-intercontinental;;;46.224457;6.133673;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592840;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47319;
+47320;Genève Jardin Botanique;geneve-jardin-botanique;;;46.22389;6.147714;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592843;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47320;
+47321;Genève Jean-Jacques;geneve-jean-jacques;;;46.204312;6.125205;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592844;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47321;
+47322;Genève Joli-Mont;geneve-joli-mont;;;46.221868;6.117141;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592845;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47322;
+47323;Genève Krieg;geneve-krieg;;;46.193767;6.160802;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592846;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47323;
+47324;Genève La Fenêtre;geneve-la-fenetre;;;46.230021;6.138734;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592847;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47324;
+47325;Genève Les Ouches;geneve-les-ouches;;;46.210209;6.117195;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8595020;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47325;
+47326;Genève Liszt;geneve-liszt;;;46.198837;6.149296;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592828;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47326;
+47327;Genève Lombard;geneve-lombard;;;46.19552;6.14721;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592848;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47327;
+47328;Genève Musée d’art et d’histoire;geneve-musee-dart-et-dhistoire;;;46.199628;6.151139;5335;CH;Europe/Zurich;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;8592851;t;;f;;f;;f;;f;;;;;f;;f;;f;;f;;f;;f;f;;Genf;Geneva;Ginebra;;Ginevra;;;Ženeva;;Genf;ジュネーヴ;제네바;Genewa;Genebra;Женева;;Cenevre;日内瓦;urn:trainline:public:nloc:csv47328;
```

Comment:
:thinking: Shouldn't `’` be treated as a separator for `slug`?
I would prefer rather `geneve-musee-d-art-et-d-histoire` than `geneve-musee-dart-et-dhistoire`, what do you think?

Reply:
If I make `’` slugify into `-` we have 436 slugs to change
```
Station Aire-sur-l’Adour (8) has an incorrect slug, "aire-sur-ladour" should match /^aire-sur-l-adour-?(bis|gare|fr)?$/
Station L’Isthme (112) has an incorrect slug, "listhme" should match /^l-isthme-?(bis|gare|fr)?$/
Station Croix-l’Allumette (131) has an incorrect slug, "croix-lallumette" should match /^croix-l-allumette-?(bis|gare|fr)?$/
Station Raon-l’Étape (553) has an incorrect slug, "raon-letape" should match /^raon-l-etape-?(bis|gare|fr)?$/
Station Bagnoles-de-l’Orne Office de Tourisme (579) has an incorrect slug, "bagnoles-de-lorne-office-de-tourisme" should match /^bagnoles-de-l-orne-office-de-tourisme-?(bis|gare|fr)?$/
Station Bagnoles-de-l’Orne (654) has an incorrect slug, "bagnoles-de-lorne" should match /^bagnoles-de-l-orne-?(bis|gare|fr)?$/
Station L’Isle-sur-le-Doubs (789) has an incorrect slug, "lisle-sur-le-doubs" should match /^l-isle-sur-le-doubs-?(bis|gare|fr)?$/
Station Pont-d’Ardres (1272) has an incorrect slug, "pont-dardres" should match /^pont-d-ardres-?(bis|gare|fr)?$/
Station Chéry-les-Rozoy Place de l’Église (1396) has an incorrect slug, "chery-les-rozoy-place-de-leglise" should match /^chery-les-rozoy-place-de-l-eglise-?(bis|gare|fr)?$/
Station St-Rémy-en-l’Eau (1433) has an incorrect slug, "st-remy-en-leau" should match /^st-remy-en-l-eau-?(bis|gare|fr)?$/
Station Clermont-de-l’Oise (1435) has an incorrect slug, "clermont-de-loise" should match /^clermont-de-l-oise-?(bis|gare|fr)?$/
Station Le Château-d’Oléron (1487) has an incorrect slug, "le-chateau-doleron" should match /^le-chateau-d-oleron-?(bis|gare|fr)?$/
Station Buire-sur-l’Ancre (1493) has an incorrect slug, "buire-sur-lancre" should match /^buire-sur-l-ancre-?(bis|gare|fr)?$/
Station Conflans-Fin-d’Oise (1691) has an incorrect slug, "conflans-fin-doise" should match /^conflans-fin-d-oise-?(bis|gare|fr)?$/
Station Dolus-d’Oléron (1908) has an incorrect slug, "dolus-doleron" should match /^dolus-d-oleron-?(bis|gare|fr)?$/
Station St-Vaast-d’Équiqueville (1930) has an incorrect slug, "st-vaast-dequiqueville" should match /^st-vaast-d-equiqueville-?(bis|gare|fr)?$/
Station Pont-de-l’Arche (1946) has an incorrect slug, "pont-de-larche" should match /^pont-de-l-arche-?(bis|gare|fr)?$/
Station L’Aigle (1981) has an incorrect slug, "laigle" should match /^l-aigle-?(bis|gare|fr)?$/
Station L’Hermitage – Mordelles (2016) has an incorrect slug, "lhermitage-mordelles" should match /^l-hermitage-mordelles-?(bis|gare|fr)?$/
Station Plœuc – L’Hermitage (2066) has an incorrect slug, "ploeuc-lhermitage" should match /^ploeuc-l-hermitage-?(bis|gare|fr)?$/
Station Ste-Anne-d’Auray (2145) has an incorrect slug, "ste-anne-dauray" should match /^ste-anne-d-auray-?(bis|gare|fr)?$/
Station L’Île-d’Elle (2231) has an incorrect slug, "lile-delle" should match /^l-ile-d-elle-?(bis|gare|fr)?$/
Station L’Herbergement – Les-Brouzils (2253) has an incorrect slug, "lherbergement-les-brouzils" should match /^l-herbergement-les-brouzils-?(bis|gare|fr)?$/
Station Fontaines-d’Ozillac (2302) has an incorrect slug, "fontaines-dozillac" should match /^fontaines-d-ozillac-?(bis|gare|fr)?$/
Station La Grave-d’Ambarès (2313) has an incorrect slug, "la-grave-dambares" should match /^la-grave-d-ambares-?(bis|gare|fr)?$/
Station St-Jean-d’Angély (2317) has an incorrect slug, "st-jean-dangely" should match /^st-jean-d-angely-?(bis|gare|fr)?$/
Station L’Habitarelle (2324) has an incorrect slug, "lhabitarelle" should match /^l-habitarelle-?(bis|gare|fr)?$/
Station Notre-Dame-d’Oé (2462) has an incorrect slug, "notre-dame-doe" should match /^notre-dame-d-oe-?(bis|gare|fr)?$/
Station Faubourg-d’Orléans (2512) has an incorrect slug, "faubourg-dorleans" should match /^faubourg-d-orleans-?(bis|gare|fr)?$/
Station La Guerche-sur-l’Aubois (2533) has an incorrect slug, "la-guerche-sur-laubois" should match /^la-guerche-sur-l-aubois-?(bis|gare|fr)?$/
Station Villenave-d’Ornon (2568) has an incorrect slug, "villenave-dornon" should match /^villenave-d-ornon-?(bis|gare|fr)?$/
Station Croix-d’Hins (2598) has an incorrect slug, "croix-dhins" should match /^croix-d-hins-?(bis|gare|fr)?$/
Station St-Médard-d’Eyrans (2609) has an incorrect slug, "st-medard-deyrans" should match /^st-medard-d-eyrans-?(bis|gare|fr)?$/
Station St-Pierre-d’Aurillac (2619) has an incorrect slug, "st-pierre-daurillac" should match /^st-pierre-d-aurillac-?(bis|gare|fr)?$/
Station St-Seurin-sur-l’Isle (2656) has an incorrect slug, "st-seurin-sur-lisle" should match /^st-seurin-sur-l-isle-?(bis|gare|fr)?$/
Station L’Aiguille (2717) has an incorrect slug, "laiguille" should match /^l-aiguille-?(bis|gare|fr)?$/
Station Le Mas-d’Artige (2771) has an incorrect slug, "le-mas-dartige" should match /^le-mas-d-artige-?(bis|gare|fr)?$/
Station Château-l’Évêque (2774) has an incorrect slug, "chateau-leveque" should match /^chateau-l-eveque-?(bis|gare|fr)?$/
Station St-Léon-sur-l’Isle (2780) has an incorrect slug, "st-leon-sur-lisle" should match /^st-leon-sur-l-isle-?(bis|gare|fr)?$/
Station Milhac-d’Auberoche (2788) has an incorrect slug, "milhac-dauberoche" should match /^milhac-d-auberoche-?(bis|gare|fr)?$/
Station Vernet-d’Ariège (2870) has an incorrect slug, "vernet-dariege" should match /^vernet-d-ariege-?(bis|gare|fr)?$/
Station L’Hospitalet-près-l’Andorre (2882) has an incorrect slug, "lhospitalet-pres-landorre" should match /^l-hospitalet-pres-l-andorre-?(bis|gare|fr)?$/
Station Castelnau-d’Estrétefonds (2893) has an incorrect slug, "castelnau-destretefonds" should match /^castelnau-d-estretefonds-?(bis|gare|fr)?$/
Station L’Isle-Jourdain (2905) has an incorrect slug, "lisle-jourdain" should match /^l-isle-jourdain-?(bis|gare|fr)?$/
Station Valence-d’Agen (2911) has an incorrect slug, "valence-dagen" should match /^valence-d-agen-?(bis|gare|fr)?$/
Station Puy-l’Évêque (2971) has an incorrect slug, "puy-leveque" should match /^puy-l-eveque-?(bis|gare|fr)?$/
Station Villeneuve-d’Aveyron (2977) has an incorrect slug, "villeneuve-daveyron" should match /^villeneuve-d-aveyron-?(bis|gare|fr)?$/
Station Laroque-d’Olmes (3014) has an incorrect slug, "laroque-dolmes" should match /^laroque-d-olmes-?(bis|gare|fr)?$/
Station St-Sauves-d’Auvergne (3101) has an incorrect slug, "st-sauves-dauvergne" should match /^st-sauves-d-auvergne-?(bis|gare|fr)?$/
Station Grenade-sur-l’Adour (3134) has an incorrect slug, "grenade-sur-ladour" should match /^grenade-sur-l-adour-?(bis|gare|fr)?$/
Station Cazères-sur-l’Adour (3135) has an incorrect slug, "cazeres-sur-ladour" should match /^cazeres-sur-l-adour-?(bis|gare|fr)?$/
Station St-Martin-d’Oney (3163) has an incorrect slug, "st-martin-doney" should match /^st-martin-d-oney-?(bis|gare|fr)?$/
Station Forges-d’Abel (3234) has an incorrect slug, "forges-dabel" should match /^forges-d-abel-?(bis|gare|fr)?$/
Station L’Orme – St-Pantaléon (3433) has an incorrect slug, "lorme-st-pantaleon" should match /^l-orme-st-pantaleon-?(bis|gare|fr)?$/
Station Paris-L’Hôpital (3497) has an incorrect slug, "paris-lhopital" should match /^paris-l-hopital-?(bis|gare|fr)?$/
Station Pont-d’Esquit (3515) has an incorrect slug, "pont-desquit" should match /^pont-d-esquit-?(bis|gare|fr)?$/
Station Collonges (Côte d’Or) (3539) has an incorrect slug, "collonges-cote-dor" should match /^collonges-cote-d-or-?(bis|gare|fr)?$/
Station L’Hôpital-du-Gros-Bois (3632) has an incorrect slug, "lhopital-du-gros-bois" should match /^l-hopital-du-gros-bois-?(bis|gare|fr)?$/
Station Couzon-au-Mont-d’Or (3656) has an incorrect slug, "couzon-au-mont-dor" should match /^couzon-au-mont-d-or-?(bis|gare|fr)?$/
Station Fleurieux-sur-l’Arbresle (3672) has an incorrect slug, "fleurieux-sur-larbresle" should match /^fleurieux-sur-l-arbresle-?(bis|gare|fr)?$/
Station Civrieux-d’Azergues (3676) has an incorrect slug, "civrieux-dazergues" should match /^civrieux-d-azergues-?(bis|gare|fr)?$/
Station Châtillon-d’Azergues (3677) has an incorrect slug, "chatillon-dazergues" should match /^chatillon-d-azergues-?(bis|gare|fr)?$/
Station Bois-d’Oingt – Légny (3680) has an incorrect slug, "bois-doingt-legny" should match /^bois-d-oingt-legny-?(bis|gare|fr)?$/
Station St-Nizier-d’Azergues (3687) has an incorrect slug, "st-nizier-dazergues" should match /^st-nizier-d-azergues-?(bis|gare|fr)?$/
Station Bois-d’Oingt Centre (3690) has an incorrect slug, "bois-doingt-centre" should match /^bois-d-oingt-centre-?(bis|gare|fr)?$/
Station Ceaux-d’Allègre (3809) has an incorrect slug, "ceaux-dallegre" should match /^ceaux-d-allegre-?(bis|gare|fr)?$/
Station Monistrol-d’Allier (3843) has an incorrect slug, "monistrol-dallier" should match /^monistrol-d-allier-?(bis|gare|fr)?$/
Station St-Loup Place de l’Église (3885) has an incorrect slug, "st-loup-place-de-leglise" should match /^st-loup-place-de-l-eglise-?(bis|gare|fr)?$/
Station Pont-d’Ain (3950) has an incorrect slug, "pont-dain" should match /^pont-d-ain-?(bis|gare|fr)?$/
Station L’Ariane – La Trinité (4115) has an incorrect slug, "lariane-la-trinite" should match /^l-ariane-la-trinite-?(bis|gare|fr)?$/
Station Cap-d’Ail (4167) has an incorrect slug, "cap-dail" should match /^cap-d-ail-?(bis|gare|fr)?$/
Station L’Escarène (4175) has an incorrect slug, "lescarene" should match /^l-escarene-?(bis|gare|fr)?$/
Station Touët-de-l’Escarène (4176) has an incorrect slug, "touet-de-lescarene" should match /^touet-de-l-escarene-?(bis|gare|fr)?$/
Station St-Rambert-d’Albon (4269) has an incorrect slug, "st-rambert-dalbon" should match /^st-rambert-d-albon-?(bis|gare|fr)?$/
Station Tain-L’Hermitage – Tournon (4272) has an incorrect slug, "tain-lhermitage-tournon" should match /^tain-l-hermitage-tournon-?(bis|gare|fr)?$/
Station L’Isle-d’Abeau (4274) has an incorrect slug, "lisle-dabeau" should match /^l-isle-d-abeau-?(bis|gare|fr)?$/
Station L’Argentière – Les-Ecrins (4328) has an incorrect slug, "largentiere-les-ecrins" should match /^l-argentiere-les-ecrins-?(bis|gare|fr)?$/
Station St-Saturnin-d’Avignon (4353) has an incorrect slug, "st-saturnin-davignon" should match /^st-saturnin-d-avignon-?(bis|gare|fr)?$/
Station L’Isle-sur-Sorgue – Fontaine-de-Vaucluse (4356) has an incorrect slug, "lisle-sur-sorgue-fontaine-de-vaucluse" should match /^l-isle-sur-sorgue-fontaine-de-vaucluse-?(bis|gare|fr)?$/
Station Ste-Cécile-d’Andorge (4385) has an incorrect slug, "ste-cecile-dandorge" should match /^ste-cecile-d-andorge-?(bis|gare|fr)?$/
Station St-Laurent-d’Aigouze (4409) has an incorrect slug, "st-laurent-daigouze" should match /^st-laurent-d-aigouze-?(bis|gare|fr)?$/
Station Le Bousquet-d’Orb (4428) has an incorrect slug, "le-bousquet-dorb" should match /^le-bousquet-d-orb-?(bis|gare|fr)?$/
Station Aire-sur-l’Adour Office de Tourisme (4434) has an incorrect slug, "aire-sur-ladour-office-de-tourisme" should match /^aire-sur-l-adour-office-de-tourisme-?(bis|gare|fr)?$/
Station St-Chély-d’Apcher (4443) has an incorrect slug, "st-chely-dapcher" should match /^st-chely-d-apcher-?(bis|gare|fr)?$/
Station St-Féliu-d’Avail (4476) has an incorrect slug, "st-feliu-davail" should match /^st-feliu-d-avail-?(bis|gare|fr)?$/
Station Conflans-Fin-d’Oise TGV (4578) has an incorrect slug, "conflans-fin-doise-tgv" should match /^conflans-fin-d-oise-tgv-?(bis|gare|fr)?$/
Station L’Arbresle (4607) has an incorrect slug, "larbresle" should match /^l-arbresle-?(bis|gare|fr)?$/
Station L’Estaque (4633) has an incorrect slug, "lestaque" should match /^l-estaque-?(bis|gare|fr)?$/
Station Les Sables-d’Olonne (4698) has an incorrect slug, "les-sables-dolonne" should match /^les-sables-d-olonne-?(bis|gare|fr)?$/
Station Lalevade-d’Ardèche (4713) has an incorrect slug, "lalevade-dardeche" should match /^lalevade-d-ardeche-?(bis|gare|fr)?$/
Station Noirmoutier-en-l’Île (4859) has an incorrect slug, "noirmoutier-en-lile" should match /^noirmoutier-en-l-ile-?(bis|gare|fr)?$/
Station Ossès – St-Martin-d’Arrossa (4909) has an incorrect slug, "osses-st-martin-darrossa" should match /^osses-st-martin-d-arrossa-?(bis|gare|fr)?$/
Station Port-d’Atelier-Amance (4914) has an incorrect slug, "port-datelier-amance" should match /^port-d-atelier-amance-?(bis|gare|fr)?$/
Station Paris Bercy Bourgogne-Pays d’Auvergne (4917) has an incorrect slug, "paris-bercy-bourgogne-pays-dauvergne" should match /^paris-bercy-bourgogne-pays-d-auvergne-?(bis|gare|fr)?$/
Station Paris Gare de l’Est (4919) has an incorrect slug, "paris-gare-de-lest" should match /^paris-gare-de-l-est-?(bis|gare|fr)?$/
Station St-Pierre-d’Oléron (4984) has an incorrect slug, "st-pierre-doleron" should match /^st-pierre-d-oleron-?(bis|gare|fr)?$/
Station Pont-l’Évêque (5011) has an incorrect slug, "pont-leveque" should match /^pont-l-eveque-?(bis|gare|fr)?$/
Station Miramont-d’Astarac (5026) has an incorrect slug, "miramont-dastarac" should match /^miramont-d-astarac-?(bis|gare|fr)?$/
Station Reims Franchet d’Esperey (5170) has an incorrect slug, "reims-franchet-desperey" should match /^reims-franchet-d-esperey-?(bis|gare|fr)?$/
Station St-Georges-d’Aurac (5178) has an incorrect slug, "st-georges-daurac" should match /^st-georges-d-aurac-?(bis|gare|fr)?$/
Station St-Georges-d’Aurac (5179) has an incorrect slug, "st-georges-daurac" should match /^st-georges-d-aurac-?(bis|gare|fr)?$/
Station St-Germain-au-Mont-d’Or (5184) has an incorrect slug, "st-germain-au-mont-dor" should match /^st-germain-au-mont-d-or-?(bis|gare|fr)?$/
Station St-Pierre-d’Albigny (5228) has an incorrect slug, "st-pierre-dalbigny" should match /^st-pierre-d-albigny-?(bis|gare|fr)?$/
Station Toulouse Jeanne d’Arc (5312) has an incorrect slug, "toulouse-jeanne-darc" should match /^toulouse-jeanne-d-arc-?(bis|gare|fr)?$/
Station Vabres-l’Abbaye (5386) has an incorrect slug, "vabres-labbaye" should match /^vabres-l-abbaye-?(bis|gare|fr)?$/
Station Villefranche d’Albigeois (5392) has an incorrect slug, "villefranche-dalbigeois" should match /^villefranche-d-albigeois-?(bis|gare|fr)?$/
Station Carcassonne Pont d’Artigues (5396) has an incorrect slug, "carcassonne-pont-dartigues" should match /^carcassonne-pont-d-artigues-?(bis|gare|fr)?$/
Station Labastide-d’Anjou Poste (5403) has an incorrect slug, "labastide-danjou-poste" should match /^labastide-d-anjou-poste-?(bis|gare|fr)?$/
Station L’Huis-Renaud (5435) has an incorrect slug, "lhuis-renaud" should match /^l-huis-renaud-?(bis|gare|fr)?$/
Station Haut-d’Espoey (5450) has an incorrect slug, "haut-despoey" should match /^haut-d-espoey-?(bis|gare|fr)?$/
Station Ste-Christie-d’Armagnac (5503) has an incorrect slug, "ste-christie-darmagnac" should match /^ste-christie-d-armagnac-?(bis|gare|fr)?$/
Station Vallon-Pont-d’Arc (5668) has an incorrect slug, "vallon-pont-darc" should match /^vallon-pont-d-arc-?(bis|gare|fr)?$/
Station Sant’Antonio (6134) has an incorrect slug, "santantonio" should match /^sant-antonio-?(bis|bahnhof|gare|stazione|ch)?$/
Station L’Arboç (6611) has an incorrect slug, "larboc" should match /^l-arboc-?(bis|estacion|ciudad|es)?$/
Station Morhange Jeanne d’Arc (8797) has an incorrect slug, "morhange-jeanne-darc" should match /^morhange-jeanne-d-arc-?(bis|gare|fr)?$/
Station Le Lion-d’Angers (8817) has an incorrect slug, "le-lion-dangers" should match /^le-lion-d-angers-?(bis|gare|fr)?$/
Station Rosiers-d’Égletons (8830) has an incorrect slug, "rosiers-degletons" should match /^rosiers-d-egletons-?(bis|gare|fr)?$/
Station Puy-l’Évêque Place Rampeau (8878) has an incorrect slug, "puy-leveque-place-rampeau" should match /^puy-l-eveque-place-rampeau-?(bis|gare|fr)?$/
Station Puy-l’Évêque Place Platane (8879) has an incorrect slug, "puy-leveque-place-platane" should match /^puy-l-eveque-place-platane-?(bis|gare|fr)?$/
Station L’Isle-Jourdain Caserne (8894) has an incorrect slug, "lisle-jourdain-caserne" should match /^l-isle-jourdain-caserne-?(bis|gare|fr)?$/
Station Le Puy-en-Velay l’Hermitage (8920) has an incorrect slug, "le-puy-en-velay-lhermitage" should match /^le-puy-en-velay-l-hermitage-?(bis|gare|fr)?$/
Station Urçay Hôtel du Lion d’Or (8945) has an incorrect slug, "urcay-hotel-du-lion-dor" should match /^urcay-hotel-du-lion-d-or-?(bis|gare|fr)?$/
Station Niederbronn Maison de l’Archéologie (8986) has an incorrect slug, "niederbronn-maison-de-larcheologie" should match /^niederbronn-maison-de-l-archeologie-?(bis|gare|fr)?$/
Station Villefranche-d’Allier (9017) has an incorrect slug, "villefranche-dallier" should match /^villefranche-d-allier-?(bis|gare|fr)?$/
Station St-Denis-d’Oléron (9032) has an incorrect slug, "st-denis-doleron" should match /^st-denis-d-oleron-?(bis|gare|fr)?$/
Station Aéroport-d’Orly-Val (9038) has an incorrect slug, "aeroport-dorly-val" should match /^aeroport-d-orly-val-?(bis|gare|fr)?$/
Station Labastide-d’Anjou Hôtel (9066) has an incorrect slug, "labastide-danjou-hotel" should match /^labastide-d-anjou-hotel-?(bis|gare|fr)?$/
Station Grand’Combe-Châteleu (9094) has an incorrect slug, "grandcombe-chateleu" should match /^grand-combe-chateleu-?(bis|gare|fr)?$/
Station Val-d’Europe (9096) has an incorrect slug, "val-deurope" should match /^val-d-europe-?(bis|gare|fr)?$/
Station Montfort-l’Amaury – Méré (9262) has an incorrect slug, "montfort-lamaury-mere" should match /^montfort-l-amaury-mere-?(bis|gare|fr)?$/
Station Val-d’Isère (9327) has an incorrect slug, "val-disere" should match /^val-d-isere-?(bis|gare|fr)?$/
Station Aéroport-d’Orly (9380) has an incorrect slug, "aeroport-dorly" should match /^aeroport-d-orly-?(bis|gare|fr)?$/
Station St-Julien-l’Ars (9381) has an incorrect slug, "st-julien-lars" should match /^st-julien-l-ars-?(bis|gare|fr)?$/
Station Pont-de-l’Alma (9388) has an incorrect slug, "pont-de-lalma" should match /^pont-de-l-alma-?(bis|gare|fr)?$/
Station Puy-l’Évêque (9503) has an incorrect slug, "puy-leveque" should match /^puy-l-eveque-?(bis|gare|fr)?$/
Station Benevent-l’Abbaye (9520) has an incorrect slug, "benevent-labbaye" should match /^benevent-l-abbaye-?(bis|gare|fr)?$/
Station Sèvres-Ville-d’Avray (9566) has an incorrect slug, "sevres-ville-davray" should match /^sevres-ville-d-avray-?(bis|gare|fr)?$/
Station Le Val-d’Or (9570) has an incorrect slug, "le-val-dor" should match /^le-val-d-or-?(bis|gare|fr)?$/
Station L’Étang-la-Ville (9579) has an incorrect slug, "letang-la-ville" should match /^l-etang-la-ville-?(bis|gare|fr)?$/
Station Coatloc’h (9601) has an incorrect slug, "coatloch" should match /^coatloc-h-?(bis|gare|fr)?$/
Station Mittersheim l’Escale (9608) has an incorrect slug, "mittersheim-lescale" should match /^mittersheim-l-escale-?(bis|gare|fr)?$/
Station St-Leu-d’Esserent (9621) has an incorrect slug, "st-leu-desserent" should match /^st-leu-d-esserent-?(bis|gare|fr)?$/
Station L’Isle-Adam – Parmain (9635) has an incorrect slug, "lisle-adam-parmain" should match /^l-isle-adam-parmain-?(bis|gare|fr)?$/
Station St-Ouen-l’Aumône Église (9646) has an incorrect slug, "st-ouen-laumone-eglise" should match /^st-ouen-l-aumone-eglise-?(bis|gare|fr)?$/
Station Val-d’Argenteuil (9668) has an incorrect slug, "val-dargenteuil" should match /^val-d-argenteuil-?(bis|gare|fr)?$/
Station Nogent-l’Artaud – Charly (9694) has an incorrect slug, "nogent-lartaud-charly" should match /^nogent-l-artaud-charly-?(bis|gare|fr)?$/
Station Champ-de-Courses-d’Enghien (9732) has an incorrect slug, "champ-de-courses-denghien" should match /^champ-de-courses-d-enghien-?(bis|gare|fr)?$/
Station St-Ouen-l’Aumône (9739) has an incorrect slug, "st-ouen-laumone" should match /^st-ouen-l-aumone-?(bis|gare|fr)?$/
Station Boissy-l’Aillerie (9759) has an incorrect slug, "boissy-laillerie" should match /^boissy-l-aillerie-?(bis|gare|fr)?$/
Station Verneuil-l’Étang (9784) has an incorrect slug, "verneuil-letang" should match /^verneuil-l-etang-?(bis|gare|fr)?$/
Station St-Georges-d’Oléron (9806) has an incorrect slug, "st-georges-doleron" should match /^st-georges-d-oleron-?(bis|gare|fr)?$/
Station Pont-l’Évêque-sur-Oise (9834) has an incorrect slug, "pont-leveque-sur-oise" should match /^pont-l-eveque-sur-oise-?(bis|gare|fr)?$/
Station L’Abbaye (9853) has an incorrect slug, "labbaye" should match /^l-abbaye-?(bis|gare|fr)?$/
Station Bagnoles-de-l’Orne (9871) has an incorrect slug, "bagnoles-de-lorne" should match /^bagnoles-de-l-orne-?(bis|gare|fr)?$/
Station Bagnoles-de-l’Orne Église (9878) has an incorrect slug, "bagnoles-de-lorne-eglise" should match /^bagnoles-de-l-orne-eglise-?(bis|gare|fr)?$/
Station Musée-d’Orsay (9891) has an incorrect slug, "musee-dorsay" should match /^musee-d-orsay-?(bis|gare|fr)?$/
Station St-Jean-d’Heurs (9963) has an incorrect slug, "st-jean-dheurs" should match /^st-jean-d-heurs-?(bis|gare|fr)?$/
Station St-Georges-d’Aurac Bourg (9972) has an incorrect slug, "st-georges-daurac-bourg" should match /^st-georges-d-aurac-bourg-?(bis|gare|fr)?$/
Station Sant’Ambroggio (9990) has an incorrect slug, "santambroggio" should match /^sant-ambroggio-?(bis|gare|fr)?$/
Station L’Arinella (9998) has an incorrect slug, "larinella" should match /^l-arinella-?(bis|gare|fr)?$/
Station Pont-de-Rungis-Aéroport-d’Orly (10000) has an incorrect slug, "pont-de-rungis-aeroport-dorly" should match /^pont-de-rungis-aeroport-d-orly-?(bis|gare|fr)?$/
Station Chemin-d’Antony (10010) has an incorrect slug, "chemin-dantony" should match /^chemin-d-antony-?(bis|gare|fr)?$/
Station St-Pierre-d’Argenton (10011) has an incorrect slug, "st-pierre-dargenton" should match /^st-pierre-d-argenton-?(bis|gare|fr)?$/
Station L’Hôpital-sur-Rhins (10038) has an incorrect slug, "lhopital-sur-rhins" should match /^l-hopital-sur-rhins-?(bis|gare|fr)?$/
Station St-Martin-d’Étampes (10126) has an incorrect slug, "st-martin-detampes" should match /^st-martin-d-etampes-?(bis|gare|fr)?$/
Station Orangis-Bois-de-l’Épine (10159) has an incorrect slug, "orangis-bois-de-lepine" should match /^orangis-bois-de-l-epine-?(bis|gare|fr)?$/
Station Wiencourt-l’Équipée (10206) has an incorrect slug, "wiencourt-lequipee" should match /^wiencourt-l-equipee-?(bis|gare|fr)?$/
Station Nieul-sur-l’Autise (10211) has an incorrect slug, "nieul-sur-lautise" should match /^nieul-sur-l-autise-?(bis|gare|fr)?$/
Station Noisy-Le-Grand – Mont-d’Est (10305) has an incorrect slug, "noisy-le-grand-mont-dest" should match /^noisy-le-grand-mont-d-est-?(bis|gare|fr)?$/
Station L’Hospitalet (10387) has an incorrect slug, "lhospitalet" should match /^l-hospitalet-?(bis|gare|fr)?$/
Station St-Laurent-d’Oingt (10412) has an incorrect slug, "st-laurent-doingt" should match /^st-laurent-d-oingt-?(bis|gare|fr)?$/
Station St-Just-d’Avray (10413) has an incorrect slug, "st-just-davray" should match /^st-just-d-avray-?(bis|gare|fr)?$/
Station Hippodrome-de-la-Côte-d’Azur (10423) has an incorrect slug, "hippodrome-de-la-cote-dazur" should match /^hippodrome-de-la-cote-d-azur-?(bis|gare|fr)?$/
Station L’Escale (10438) has an incorrect slug, "lescale" should match /^l-escale-?(bis|gare|fr)?$/
Station Louviers Porte de l’eau (10500) has an incorrect slug, "louviers-porte-de-leau" should match /^louviers-porte-de-l-eau-?(bis|gare|fr)?$/
Station Tannay Place de l’Église (10597) has an incorrect slug, "tannay-place-de-leglise" should match /^tannay-place-de-l-eglise-?(bis|gare|fr)?$/
Station Aire-sur-l’Adour (10607) has an incorrect slug, "aire-sur-ladour" should match /^aire-sur-l-adour-?(bis|gare|fr)?$/
Station St Ouen l’Aumône Liesse (10647) has an incorrect slug, "st-ouen-laumone-liesse" should match /^st-ouen-l-aumone-liesse-?(bis|gare|fr)?$/
Station Vernet d’Ariège (10664) has an incorrect slug, "vernet-dariege" should match /^vernet-d-ariege-?(bis|gare|fr)?$/
Station Bois d’Oingt (10692) has an incorrect slug, "bois-doingt" should match /^bois-d-oingt-?(bis|gare|fr)?$/
Station Bayard Route d’Eurville (10695) has an incorrect slug, "bayard-route-deurville" should match /^bayard-route-d-eurville-?(bis|gare|fr)?$/
Station Vernet d’Ariège Abri Bus (10763) has an incorrect slug, "vernet-dariege-abri-bus" should match /^vernet-d-ariege-abri-bus-?(bis|gare|fr)?$/
Station Pussay Château d’Eau (10768) has an incorrect slug, "pussay-chateau-deau" should match /^pussay-chateau-d-eau-?(bis|gare|fr)?$/
Station CEM Centre d’Exploitation et de Maintenance (10790) has an incorrect slug, "cem-centre-dexploitation-et-de-maintenance" should match /^cem-centre-d-exploitation-et-de-maintenance-?(bis|gare|fr)?$/
Station Dore l’Église (16828) has an incorrect slug, "dore-leglise" should match /^dore-l-eglise-?(bis|gare|fr)?$/
Station Ceint-d’Eau Village (16859) has an incorrect slug, "ceint-deau-village" should match /^ceint-d-eau-village-?(bis|gare|fr)?$/
Station L’Hôpital Village (16861) has an incorrect slug, "lhopital-village" should match /^l-hopital-village-?(bis|gare|fr)?$/
Station Saint-Martin d’Hères (17010) has an incorrect slug, "saint-martin-dheres" should match /^saint-martin-d-heres-?(bis|gare|fr)?$/
Station Noisy le Grand Mont d’Est RER (17034) has an incorrect slug, "noisy-le-grand-mont-dest-rer" should match /^noisy-le-grand-mont-d-est-rer-?(bis|gare|fr)?$/
Station Val d’Europe Serris Montévrain Noctilien (17049) has an incorrect slug, "val-deurope-serris-montevrain-noctilien" should match /^val-d-europe-serris-montevrain-noctilien-?(bis|gare|fr)?$/
Station Conflans-Fin-d’Oise Noctilien (17086) has an incorrect slug, "conflans-fin-doise-noctilien" should match /^conflans-fin-d-oise-noctilien-?(bis|gare|fr)?$/
Station Aerogare d’Orly Sud (17096) has an incorrect slug, "aerogare-dorly-sud" should match /^aerogare-d-orly-sud-?(bis|gare|fr)?$/
Station Place d’Italie (17111) has an incorrect slug, "place-ditalie" should match /^place-d-italie-?(bis|gare|fr)?$/
Station Porte d’Italie (17112) has an incorrect slug, "porte-ditalie" should match /^porte-d-italie-?(bis|gare|fr)?$/
Station Labastide d’Armagnac (17158) has an incorrect slug, "labastide-darmagnac" should match /^labastide-d-armagnac-?(bis|gare|fr)?$/
Station Saint-Sauves-d’Auvergne Bourg (17225) has an incorrect slug, "saint-sauves-dauvergne-bourg" should match /^saint-sauves-d-auvergne-bourg-?(bis|gare|fr)?$/
Station Château d’Eau (17249) has an incorrect slug, "chateau-deau" should match /^chateau-d-eau-?(bis|gare|fr)?$/
Station Le Fief d’Artois (17259) has an incorrect slug, "le-fief-dartois" should match /^le-fief-d-artois-?(bis|gare|fr)?$/
Station Gaillac d’Aveyron RN D (17288) has an incorrect slug, "gaillac-daveyron-rn-d" should match /^gaillac-d-aveyron-rn-d-?(bis|gare|fr)?$/
Station Sévérac-l’Église Embranchement (17294) has an incorrect slug, "severac-leglise-embranchement" should match /^severac-l-eglise-embranchement-?(bis|gare|fr)?$/
Station L’Hermet (17341) has an incorrect slug, "lhermet" should match /^l-hermet-?(bis|gare|fr)?$/
Station Vigneux Patt d’Oie (17354) has an incorrect slug, "vigneux-patt-doie" should match /^vigneux-patt-d-oie-?(bis|gare|fr)?$/
Station Voray-sur-l’Ognon (17361) has an incorrect slug, "voray-sur-lognon" should match /^voray-sur-l-ognon-?(bis|gare|fr)?$/
Station Lavilledieu Zone d’activité (17367) has an incorrect slug, "lavilledieu-zone-dactivite" should match /^lavilledieu-zone-d-activite-?(bis|gare|fr)?$/
Station Le Cygne d’Enghien (17386) has an incorrect slug, "le-cygne-denghien" should match /^le-cygne-d-enghien-?(bis|gare|fr)?$/
Station Braine-l’Alleud (17763) has an incorrect slug, "braine-lalleud" should match /^braine-l-alleud-?(bis|station|gare|be)?$/
Station Vivier d’Oie (18137) has an incorrect slug, "vivier-doie" should match /^vivier-d-oie-?(bis|station|gare|be)?$/
Station Chateau-d’Oex (18200) has an incorrect slug, "chateau-doex" should match /^chateau-d-oex-?(bis|bahnhof|gare|stazione|ch)?$/
Station Chateau-d’Oex TCPM/TPBM (18399) has an incorrect slug, "chateau-doex-tcpm-tpbm" should match /^chateau-d-oex-tcpm-tpbm-?(bis|bahnhof|gare|stazione|ch)?$/
Station Chateau-d’Oex gare (18405) has an incorrect slug, "chateau-doex-gare" should match /^chateau-d-oex-gare-?(bis|bahnhof|gare|stazione|ch)?$/
Station Bivio d’Aurisina (19193) has an incorrect slug, "bivio-daurisina" should match /^bivio-d-aurisina-?(bis|stazione|it)?$/
Station Barberino di Val d’Elsa (19202) has an incorrect slug, "barberino-di-val-delsa" should match /^barberino-di-val-d-elsa-?(bis|stazione|it)?$/
Station Cavanella d’Adige (19247) has an incorrect slug, "cavanella-dadige" should match /^cavanella-d-adige-?(bis|stazione|it)?$/
Station Capo d’Orlando-Naso (19269) has an incorrect slug, "capo-dorlando-naso" should match /^capo-d-orlando-naso-?(bis|stazione|it)?$/
Station Castello d’Annone (19323) has an incorrect slug, "castello-dannone" should match /^castello-d-annone-?(bis|stazione|it)?$/
Station Borghetto sull’Adige (19342) has an incorrect slug, "borghetto-sulladige" should match /^borghetto-sull-adige-?(bis|stazione|it)?$/
Station Ponte d’Adige/Sigmundskron (19475) has an incorrect slug, "ponte-dadige-sigmundskron" should match /^ponte-d-adige-sigmundskron-?(bis|stazione|it)?$/
Station Pescosannita-Fragneto l’Abate (19494) has an incorrect slug, "pescosannita-fragneto-labate" should match /^pescosannita-fragneto-l-abate-?(bis|stazione|it)?$/
Station Domegliara-Sant’Ambrogio (19503) has an incorrect slug, "domegliara-santambrogio" should match /^domegliara-sant-ambrogio-?(bis|stazione|it)?$/
Station Candela Sant’Agata di Puglia (19524) has an incorrect slug, "candela-santagata-di-puglia" should match /^candela-sant-agata-di-puglia-?(bis|stazione|it)?$/
Station Porto d’Ascoli (19544) has an incorrect slug, "porto-dascoli" should match /^porto-d-ascoli-?(bis|stazione|it)?$/
Station Porto Sant’Elpidio (19546) has an incorrect slug, "porto-santelpidio" should match /^porto-sant-elpidio-?(bis|stazione|it)?$/
Station Sant’Antonino di Saluggia (19641) has an incorrect slug, "santantonino-di-saluggia" should match /^sant-antonino-di-saluggia-?(bis|stazione|it)?$/
Station Valle Dell’Aniene-Mandela-Sambuci (19684) has an incorrect slug, "valle-dellaniene-mandela-sambuci" should match /^valle-dell-aniene-mandela-sambuci-?(bis|stazione|it)?$/
Station Mosciano Sant’Angelo (19710) has an incorrect slug, "mosciano-santangelo" should match /^mosciano-sant-angelo-?(bis|stazione|it)?$/
Station Palazzolo sull’Oglio (19742) has an incorrect slug, "palazzolo-sulloglio" should match /^palazzolo-sull-oglio-?(bis|stazione|it)?$/
Station Monteroduni Sant’Eusanio (19805) has an incorrect slug, "monteroduni-santeusanio" should match /^monteroduni-sant-eusanio-?(bis|stazione|it)?$/
Station Monteroni d’Arbia (19806) has an incorrect slug, "monteroni-darbia" should match /^monteroni-d-arbia-?(bis|stazione|it)?$/
Station Monticello d’Alba (19811) has an incorrect slug, "monticello-dalba" should match /^monticello-d-alba-?(bis|stazione|it)?$/
Station Quarto d’Altino (19846) has an incorrect slug, "quarto-daltino" should match /^quarto-d-altino-?(bis|stazione|it)?$/
Station Castel d’Ario (19857) has an incorrect slug, "castel-dario" should match /^castel-d-ario-?(bis|stazione|it)?$/
Station Castel Sant’Angelo (19859) has an incorrect slug, "castel-santangelo" should match /^castel-sant-angelo-?(bis|stazione|it)?$/
Station L’Aquila (19914) has an incorrect slug, "laquila" should match /^l-aquila-?(bis|stazione|it)?$/
Station Piano d’Orta Bolognano (19961) has an incorrect slug, "piano-dorta-bolognano" should match /^piano-d-orta-bolognano-?(bis|stazione|it)?$/
Station Fabriano Ca’Maiano (20140) has an incorrect slug, "fabriano-camaiano" should match /^fabriano-ca-maiano-?(bis|stazione|it)?$/
Station Civita d’Antino-Morino (20184) has an incorrect slug, "civita-dantino-morino" should match /^civita-d-antino-morino-?(bis|stazione|it)?$/
Station Cerreto d’Esi (20209) has an incorrect slug, "cerreto-desi" should match /^cerreto-d-esi-?(bis|stazione|it)?$/
Station Sant’Antimo-Sant’Arpino (20240) has an incorrect slug, "santantimo-santarpino" should match /^sant-antimo-sant-arpino-?(bis|stazione|it)?$/
Station San Marco d’Alunzio-Torrenova (20257) has an incorrect slug, "san-marco-dalunzio-torrenova" should match /^san-marco-d-alunzio-torrenova-?(bis|stazione|it)?$/
Station Sant’Antonio Mantovano (20265) has an incorrect slug, "santantonio-mantovano" should match /^sant-antonio-mantovano-?(bis|stazione|it)?$/
Station Anzola dell’Emilia (20284) has an incorrect slug, "anzola-dellemilia" should match /^anzola-dell-emilia-?(bis|stazione|it)?$/
Station Sant’Agata di Militello (20304) has an incorrect slug, "santagata-di-militello" should match /^sant-agata-di-militello-?(bis|stazione|it)?$/
Station Sant’Agata sul Santerno (20305) has an incorrect slug, "santagata-sul-santerno" should match /^sant-agata-sul-santerno-?(bis|stazione|it)?$/
Station Sant’Alessio Siculo-Forza d’Agrò (20306) has an incorrect slug, "santalessio-siculo-forza-dagro" should match /^sant-alessio-siculo-forza-d-agro-?(bis|stazione|it)?$/
Station Sant’Ambrogio (20307) has an incorrect slug, "santambrogio" should match /^sant-ambrogio-?(bis|stazione|it)?$/
Station Sant’Andrea dello Jonio (20309) has an incorrect slug, "santandrea-dello-jonio" should match /^sant-andrea-dello-jonio-?(bis|stazione|it)?$/
Station Sant’Agapito Longano (20310) has an incorrect slug, "santagapito-longano" should match /^sant-agapito-longano-?(bis|stazione|it)?$/
Station Sant’Antonino-Vaie (20340) has an incorrect slug, "santantonino-vaie" should match /^sant-antonino-vaie-?(bis|stazione|it)?$/
Station Ripabottoni-Sant’Elia (20362) has an incorrect slug, "ripabottoni-santelia" should match /^ripabottoni-sant-elia-?(bis|stazione|it)?$/
Station Vigliano d’Asti (20438) has an incorrect slug, "vigliano-dasti" should match /^vigliano-d-asti-?(bis|stazione|it)?$/
Station Sant’Eurosia (20483) has an incorrect slug, "santeurosia" should match /^sant-eurosia-?(bis|stazione|it)?$/
Station Villanova d’Asti (20579) has an incorrect slug, "villanova-dasti" should match /^villanova-d-asti-?(bis|stazione|it)?$/
Station Sant’Ilario d’Enza (20631) has an incorrect slug, "santilario-denza" should match /^sant-ilario-d-enza-?(bis|stazione|it)?$/
Station Val d’Ala (20665) has an incorrect slug, "val-dala" should match /^val-d-ala-?(bis|stazione|it)?$/
Station Sant’Ellero (20697) has an incorrect slug, "santellero" should match /^sant-ellero-?(bis|stazione|it)?$/
Station San Damiano d’Asti (20698) has an incorrect slug, "san-damiano-dasti" should match /^san-damiano-d-asti-?(bis|stazione|it)?$/
Station Sant’Anna di Chioggia (20763) has an incorrect slug, "santanna-di-chioggia" should match /^sant-anna-di-chioggia-?(bis|stazione|it)?$/
Station Vaprio d’Agogna (20781) has an incorrect slug, "vaprio-dagogna" should match /^vaprio-d-agogna-?(bis|stazione|it)?$/
Station Boschi Sant’Anna (20817) has an incorrect slug, "boschi-santanna" should match /^boschi-sant-anna-?(bis|stazione|it)?$/
Station Monticelli d’Ongina (20931) has an incorrect slug, "monticelli-dongina" should match /^monticelli-d-ongina-?(bis|stazione|it)?$/
Station Villanova d’Arda (20951) has an incorrect slug, "villanova-darda" should match /^villanova-d-arda-?(bis|stazione|it)?$/
Station Serravalle all’Adige (20978) has an incorrect slug, "serravalle-alladige" should match /^serravalle-all-adige-?(bis|stazione|it)?$/
Station Sant’Elena-Este (20987) has an incorrect slug, "santelena-este" should match /^sant-elena-este-?(bis|stazione|it)?$/
Station Castell’Alfero (21067) has an incorrect slug, "castellalfero" should match /^castell-alfero-?(bis|stazione|it)?$/
Station Cà dell’Orbo (21079) has an incorrect slug, "ca-dellorbo" should match /^ca-dell-orbo-?(bis|stazione|it)?$/
Station Apice-Sant’Arcangelo-Bonito (21084) has an incorrect slug, "apice-santarcangelo-bonito" should match /^apice-sant-arcangelo-bonito-?(bis|stazione|it)?$/
Station Corigliano d’Otranto (21167) has an incorrect slug, "corigliano-dotranto" should match /^corigliano-d-otranto-?(bis|stazione|it)?$/
Station Cassano d’Adda (21179) has an incorrect slug, "cassano-dadda" should match /^cassano-d-adda-?(bis|stazione|it)?$/
Station Costigliole d’Asti (21188) has an incorrect slug, "costigliole-dasti" should match /^costigliole-d-asti-?(bis|stazione|it)?$/
Station Albano Sant’Alessandro (21232) has an incorrect slug, "albano-santalessandro" should match /^albano-sant-alessandro-?(bis|stazione|it)?$/
Station Pizzighettone-Ponte d’Adda (21270) has an incorrect slug, "pizzighettone-ponte-dadda" should match /^pizzighettone-ponte-d-adda-?(bis|stazione|it)?$/
Station Molini d’Isola (21353) has an incorrect slug, "molini-disola" should match /^molini-d-isola-?(bis|stazione|it)?$/
Station Perugia Sant’Anna (21359) has an incorrect slug, "perugia-santanna" should match /^perugia-sant-anna-?(bis|stazione|it)?$/
Station Isola d’Arbia (21413) has an incorrect slug, "isola-darbia" should match /^isola-d-arbia-?(bis|stazione|it)?$/
Station Montechiaro d’Asti (21422) has an incorrect slug, "montechiaro-dasti" should match /^montechiaro-d-asti-?(bis|stazione|it)?$/
Station Motta Sant’Anastasia (21446) has an incorrect slug, "motta-santanastasia" should match /^motta-sant-anastasia-?(bis|stazione|it)?$/
Station Sant’Oliva (21571) has an incorrect slug, "santoliva" should match /^sant-oliva-?(bis|stazione|it)?$/
Station Pianerottolo d’Ariano (21632) has an incorrect slug, "pianerottolo-dariano" should match /^pianerottolo-d-ariano-?(bis|stazione|it)?$/
Station Piano di Coreglia-Ponte all’Ania (21634) has an incorrect slug, "piano-di-coreglia-ponte-allania" should match /^piano-di-coreglia-ponte-all-ania-?(bis|stazione|it)?$/
Station Piraino-Sant’Angelo (21740) has an incorrect slug, "piraino-santangelo" should match /^piraino-sant-angelo-?(bis|stazione|it)?$/
Station Caslino d’Erba (21781) has an incorrect slug, "caslino-derba" should match /^caslino-d-erba-?(bis|stazione|it)?$/
Station Caranzano-Sant’Andrea (21811) has an incorrect slug, "caranzano-santandrea" should match /^caranzano-sant-andrea-?(bis|stazione|it)?$/
Station Sant’Antonio (21859) has an incorrect slug, "santantonio" should match /^sant-antonio-?(bis|stazione|it)?$/
Station Bari Sant’Andrea (21908) has an incorrect slug, "bari-santandrea" should match /^bari-sant-andrea-?(bis|stazione|it)?$/
Station Sant’Angelo dei Lombardi (21911) has an incorrect slug, "santangelo-dei-lombardi" should match /^sant-angelo-dei-lombardi-?(bis|stazione|it)?$/
Station Sant’Anna Robella (21912) has an incorrect slug, "santanna-robella" should match /^sant-anna-robella-?(bis|stazione|it)?$/
Station Ripa d’Api (21940) has an incorrect slug, "ripa-dapi" should match /^ripa-d-api-?(bis|stazione|it)?$/
Station Rocca d’Evandro-San Vittore (21942) has an incorrect slug, "rocca-devandro-san-vittore" should match /^rocca-d-evandro-san-vittore-?(bis|stazione|it)?$/
Station Rocchetta Sant’Antonio-Lacedonia (21943) has an incorrect slug, "rocchetta-santantonio-lacedonia" should match /^rocchetta-sant-antonio-lacedonia-?(bis|stazione|it)?$/
Station Rignano Sull’Arno-Reggello (21956) has an incorrect slug, "rignano-sullarno-reggello" should match /^rignano-sull-arno-reggello-?(bis|stazione|it)?$/
Station Sant’Anna (Reggio di Calabria) (21981) has an incorrect slug, "santanna-reggio-di-calabria" should match /^sant-anna-reggio-di-calabria-?(bis|stazione|it)?$/
Station San Giuliano d’Arezzo (22008) has an incorrect slug, "san-giuliano-darezzo" should match /^san-giuliano-d-arezzo-?(bis|stazione|it)?$/
Station San Polo d’Enza (22017) has an incorrect slug, "san-polo-denza" should match /^san-polo-d-enza-?(bis|stazione|it)?$/
Station Vigliano d’Abruzzo (22028) has an incorrect slug, "vigliano-dabruzzo" should match /^vigliano-d-abruzzo-?(bis|stazione|it)?$/
Station Tavernelle d’Emilia (22045) has an incorrect slug, "tavernelle-demilia" should match /^tavernelle-d-emilia-?(bis|stazione|it)?$/
Station Sant’Eusanio del Sangro (22066) has an incorrect slug, "santeusanio-del-sangro" should match /^sant-eusanio-del-sangro-?(bis|stazione|it)?$/
Station Sant’Eufemia d’Aspromonte (22072) has an incorrect slug, "santeufemia-daspromonte" should match /^sant-eufemia-d-aspromonte-?(bis|stazione|it)?$/
Station Serravalle d’Asti (22107) has an incorrect slug, "serravalle-dasti" should match /^serravalle-d-asti-?(bis|stazione|it)?$/
Station Villa Sant’Angelo (22113) has an incorrect slug, "villa-santangelo" should match /^villa-sant-angelo-?(bis|stazione|it)?$/
Station Colle Val d’Elsa (22116) has an incorrect slug, "colle-val-delsa" should match /^colle-val-d-elsa-?(bis|stazione|it)?$/
Station Sant’Ilario dello Jonio (22230) has an incorrect slug, "santilario-dello-jonio" should match /^sant-ilario-dello-jonio-?(bis|stazione|it)?$/
Station Sant’Ilario Sangro (22231) has an incorrect slug, "santilario-sangro" should match /^sant-ilario-sangro-?(bis|stazione|it)?$/
Station Sant’Eufemia di Brisighella (22302) has an incorrect slug, "santeufemia-di-brisighella" should match /^sant-eufemia-di-brisighella-?(bis|stazione|it)?$/
Station Forno d’Allione (22310) has an incorrect slug, "forno-dallione" should match /^forno-d-allione-?(bis|stazione|it)?$/
Station Sant’Anna (22359) has an incorrect slug, "santanna" should match /^sant-anna-?(bis|stazione|it)?$/
Station Arpaia Airola-Sant’Agata dei Goti (22426) has an incorrect slug, "arpaia-airola-santagata-dei-goti" should match /^arpaia-airola-sant-agata-dei-goti-?(bis|stazione|it)?$/
Station Ramazzano-Passo dell’Acqua (22441) has an incorrect slug, "ramazzano-passo-dellacqua" should match /^ramazzano-passo-dell-acqua-?(bis|stazione|it)?$/
Station Ponte d’Arbia (22449) has an incorrect slug, "ponte-darbia" should match /^ponte-d-arbia-?(bis|stazione|it)?$/
Station Guidonia-Montecelio-Sant’Angelo (22456) has an incorrect slug, "guidonia-montecelio-santangelo" should match /^guidonia-montecelio-sant-angelo-?(bis|stazione|it)?$/
Station Felegara-Sant’Andrea Bagni (22471) has an incorrect slug, "felegara-santandrea-bagni" should match /^felegara-sant-andrea-bagni-?(bis|stazione|it)?$/
Station Isola d’Asti (22487) has an incorrect slug, "isola-dasti" should match /^isola-d-asti-?(bis|stazione|it)?$/
Station Canneto Sull’Oglio (22514) has an incorrect slug, "canneto-sulloglio" should match /^canneto-sull-oglio-?(bis|stazione|it)?$/
Station Castel d’Agogna (22521) has an incorrect slug, "castel-dagogna" should match /^castel-d-agogna-?(bis|stazione|it)?$/
Station Ciano d’Enza (22523) has an incorrect slug, "ciano-denza" should match /^ciano-d-enza-?(bis|stazione|it)?$/
Station San Giovanni d’Asso (22530) has an incorrect slug, "san-giovanni-dasso" should match /^san-giovanni-d-asso-?(bis|stazione|it)?$/
Station Villanova d’Ardenghi (22537) has an incorrect slug, "villanova-dardenghi" should match /^villanova-d-ardenghi-?(bis|stazione|it)?$/
Station Sant’Angelo in Formis (22573) has an incorrect slug, "santangelo-in-formis" should match /^sant-angelo-in-formis-?(bis|stazione|it)?$/
Station Stazione per l’Alpago (22574) has an incorrect slug, "stazione-per-lalpago" should match /^stazione-per-l-alpago-?(bis|stazione|it)?$/
Station San Giuliano d’Arezzo (23062) has an incorrect slug, "san-giuliano-darezzo" should match /^san-giuliano-d-arezzo-?(bis|stazione|it)?$/
Station L’Alpe d’Huez (23132) has an incorrect slug, "lalpe-dhuez" should match /^l-alpe-d-huez-?(bis|gare|fr)?$/
Station Aire-sur-l’Adour Avenue des Pyrénées (23621) has an incorrect slug, "aire-sur-ladour-avenue-des-pyrenees" should match /^aire-sur-l-adour-avenue-des-pyrenees-?(bis|gare|fr)?$/
Station Aire-sur-l’Adour Place de la Liberté (23627) has an incorrect slug, "aire-sur-ladour-place-de-la-liberte" should match /^aire-sur-l-adour-place-de-la-liberte-?(bis|gare|fr)?$/
Station Bell-Lloc d’Urgell (23758) has an incorrect slug, "bell-lloc-durgell" should match /^bell-lloc-d-urgell-?(bis|estacion|ciudad|es)?$/
Station Duesaigües-L’Argentera (23919) has an incorrect slug, "duesaigues-largentera" should match /^duesaigues-l-argentera-?(bis|estacion|ciudad|es)?$/
Station L’Aldea-Amposta-Tortosa (24050) has an incorrect slug, "laldea-amposta-tortosa" should match /^l-aldea-amposta-tortosa-?(bis|estacion|ciudad|es)?$/
Station L’Ametlla de Mar (24051) has an incorrect slug, "lametlla-de-mar" should match /^l-ametlla-de-mar-?(bis|estacion|ciudad|es)?$/
Station L’Ampolla-El Perello-Deltebre (24052) has an incorrect slug, "lampolla-el-perello-deltebre" should match /^l-ampolla-el-perello-deltebre-?(bis|estacion|ciudad|es)?$/
Station L’Espluga de Francolí (24053) has an incorrect slug, "lespluga-de-francoli" should match /^l-espluga-de-francoli-?(bis|estacion|ciudad|es)?$/
Station L’Hospitalet de l’Infant (24054) has an incorrect slug, "lhospitalet-de-linfant" should match /^l-hospitalet-de-l-infant-?(bis|estacion|ciudad|es)?$/
Station L’Hospitalet de Llobregat (24055) has an incorrect slug, "lhospitalet-de-llobregat" should match /^l-hospitalet-de-llobregat-?(bis|estacion|ciudad|es)?$/
Station Manuel-L’Enova (24148) has an incorrect slug, "manuel-lenova" should match /^manuel-l-enova-?(bis|estacion|ciudad|es)?$/
Station Riba-Roja d’Ebre (24327) has an incorrect slug, "riba-roja-debre" should match /^riba-roja-d-ebre-?(bis|estacion|ciudad|es)?$/
Station Earl’s Court L.T. (25190) has an incorrect slug, "earls-court-l-t" should match /^earl-s-court-l-t-?(bis|station|gb)?$/
Station L’Albi (27108) has an incorrect slug, "lalbi" should match /^l-albi-?(bis|estacion|ciudad|es)?$/
Station L’Ampolla (27109) has an incorrect slug, "lampolla" should match /^l-ampolla-?(bis|estacion|ciudad|es)?$/
Station L’Hostalnou (27110) has an incorrect slug, "lhostalnou" should match /^l-hostalnou-?(bis|estacion|ciudad|es)?$/
Station La Serra d’Almos (27126) has an incorrect slug, "la-serra-dalmos" should match /^la-serra-d-almos-?(bis|estacion|ciudad|es)?$/
Station Móra d’Ebre (27187) has an incorrect slug, "mora-debre" should match /^mora-d-ebre-?(bis|estacion|ciudad|es)?$/
Station Ribera d’Ondara (27312) has an incorrect slug, "ribera-dondara" should match /^ribera-d-ondara-?(bis|estacion|ciudad|es)?$/
Station Figuerola d’Orcau (27418) has an incorrect slug, "figuerola-dorcau" should match /^figuerola-d-orcau-?(bis|estacion|ciudad|es)?$/
Station L’Espluga Calba (27422) has an incorrect slug, "lespluga-calba" should match /^l-espluga-calba-?(bis|estacion|ciudad|es)?$/
Station Bellcaire d’Urgell (27577) has an incorrect slug, "bellcaire-durgell" should match /^bellcaire-d-urgell-?(bis|estacion|ciudad|es)?$/
Station Callosa d’en Sarrià (27609) has an incorrect slug, "callosa-den-sarria" should match /^callosa-d-en-sarria-?(bis|estacion|ciudad|es)?$/
Station Castellar de N’Hug (27651) has an incorrect slug, "castellar-de-nhug" should match /^castellar-de-n-hug-?(bis|estacion|ciudad|es)?$/
Station Cortina d’Ampezzo (28961) has an incorrect slug, "cortina-dampezzo" should match /^cortina-d-ampezzo-?(bis|stazione|it)?$/
Station Mesnil-Saint-Père (Lac d’orient) (29435) has an incorrect slug, "mesnil-saint-pere-lac-dorient" should match /^mesnil-saint-pere-lac-d-orient-?(bis|gare|fr)?$/
Station Saint-Chély d’Apcher (aire de la Lozère) (29440) has an incorrect slug, "saint-chely-dapcher-aire-de-la-lozere" should match /^saint-chely-d-apcher-aire-de-la-lozere-?(bis|gare|fr)?$/
Station Bellcaire d’Empordà (32332) has an incorrect slug, "bellcaire-demporda" should match /^bellcaire-d-emporda-?(bis|estacion|ciudad|es)?$/
Station Esterri d’Àneu (32333) has an incorrect slug, "esterri-daneu" should match /^esterri-d-aneu-?(bis|estacion|ciudad|es)?$/
Station L’Alguenya (32334) has an incorrect slug, "lalguenya" should match /^l-alguenya-?(bis|estacion|ciudad|es)?$/
Station L’Avellà (32335) has an incorrect slug, "lavella" should match /^l-avella-?(bis|estacion|ciudad|es)?$/
Station L’Escala (32336) has an incorrect slug, "lescala" should match /^l-escala-?(bis|estacion|ciudad|es)?$/
Station Les Cases d’Alcanar (32337) has an incorrect slug, "les-cases-dalcanar" should match /^les-cases-d-alcanar-?(bis|estacion|ciudad|es)?$/
Station L’Espunyola (32338) has an incorrect slug, "lespunyola" should match /^l-espunyola-?(bis|estacion|ciudad|es)?$/
Station L’Hostal (32339) has an incorrect slug, "lhostal" should match /^l-hostal-?(bis|estacion|ciudad|es)?$/
Station Pla de l’Ermita (32341) has an incorrect slug, "pla-de-lermita" should match /^pla-de-l-ermita-?(bis|estacion|ciudad|es)?$/
Station S’Agaró (32342) has an incorrect slug, "sagaro" should match /^s-agaro-?(bis|estacion|ciudad|es)?$/
Station Santa Cristina d’Aro (32343) has an incorrect slug, "santa-cristina-daro" should match /^santa-cristina-d-aro-?(bis|estacion|ciudad|es)?$/
Station València d’Àneu (32344) has an incorrect slug, "valencia-daneu" should match /^valencia-d-aneu-?(bis|estacion|ciudad|es)?$/
Station Vallbona d’Anoia (32345) has an incorrect slug, "vallbona-danoia" should match /^vallbona-d-anoia-?(bis|estacion|ciudad|es)?$/
Station Vilanova de l’Aguda (32346) has an incorrect slug, "vilanova-de-laguda" should match /^vilanova-de-l-aguda-?(bis|estacion|ciudad|es)?$/
Station Khmel’nyts’kyy (32568) has an incorrect slug, "khmelnytskyy" should match /^khmel-nyts-kyy-?(bis|ua)?$/
Station Castellnou d’Oluges (32578) has an incorrect slug, "castellnou-doluges" should match /^castellnou-d-oluges-?(bis|estacion|ciudad|es)?$/
Station Castelló d’Empúries (32579) has an incorrect slug, "castello-dempuries" should match /^castello-d-empuries-?(bis|estacion|ciudad|es)?$/
Station L’Alcúdia de Crespins (32580) has an incorrect slug, "lalcudia-de-crespins" should match /^l-alcudia-de-crespins-?(bis|estacion|ciudad|es)?$/
Station L’Ametlla de Montsec (32581) has an incorrect slug, "lametlla-de-montsec" should match /^l-ametlla-de-montsec-?(bis|estacion|ciudad|es)?$/
Station La Seu d’Urgell (32582) has an incorrect slug, "la-seu-durgell" should match /^la-seu-d-urgell-?(bis|estacion|ciudad|es)?$/
Station Monestir d’Avellanes (32583) has an incorrect slug, "monestir-davellanes" should match /^monestir-d-avellanes-?(bis|estacion|ciudad|es)?$/
Station L’Asse (34733) has an incorrect slug, "lasse" should match /^l-asse-?(bis|bahnhof|gare|stazione|ch)?$/
Station Nepezzano-Piano d’Accio (34797) has an incorrect slug, "nepezzano-piano-daccio" should match /^nepezzano-piano-d-accio-?(bis|stazione|it)?$/
Station L’Aquila Sassa NSI (34798) has an incorrect slug, "laquila-sassa-nsi" should match /^l-aquila-sassa-nsi-?(bis|stazione|it)?$/
Station L’Aquila San Gregorio (34799) has an incorrect slug, "laquila-san-gregorio" should match /^l-aquila-san-gregorio-?(bis|stazione|it)?$/
Station L’Aquila Campo di Pile (34800) has an incorrect slug, "laquila-campo-di-pile" should match /^l-aquila-campo-di-pile-?(bis|stazione|it)?$/
Station St. Paul’s Bay (34815) has an incorrect slug, "st-pauls-bay" should match /^st-paul-s-bay-?(bis|mt)?$/
Station Locorotondo Lame D’Olimpia (34932) has an incorrect slug, "locorotondo-lame-dolimpia" should match /^locorotondo-lame-d-olimpia-?(bis|stazione|it)?$/
Station Manduria Piazzale Sant’Antonio (34942) has an incorrect slug, "manduria-piazzale-santantonio" should match /^manduria-piazzale-sant-antonio-?(bis|stazione|it)?$/
Station Sant’Andrea (35032) has an incorrect slug, "santandrea" should match /^sant-andrea-?(bis|stazione|it)?$/
Station Torre Dell’Orso (35065) has an incorrect slug, "torre-dellorso" should match /^torre-dell-orso-?(bis|stazione|it)?$/
Station Monte Sant’Angelo (35187) has an incorrect slug, "monte-santangelo" should match /^monte-sant-angelo-?(bis|stazione|it)?$/
Station Sant’Arcangelo (35267) has an incorrect slug, "santarcangelo" should match /^sant-arcangelo-?(bis|stazione|it)?$/
Station San Mango d’Aquino (35276) has an incorrect slug, "san-mango-daquino" should match /^san-mango-d-aquino-?(bis|stazione|it)?$/
Station Santa Sofia d’Epiro (35720) has an incorrect slug, "santa-sofia-depiro" should match /^santa-sofia-d-epiro-?(bis|stazione|it)?$/
Station La Chaux-d’Abel (36036) has an incorrect slug, "la-chaux-dabel" should match /^la-chaux-d-abel-?(bis|bahnhof|gare|stazione|ch)?$/
Station Le Pied-d’Or (36413) has an incorrect slug, "le-pied-dor" should match /^le-pied-d-or-?(bis|bahnhof|gare|stazione|ch)?$/
Station L’Isle (36622) has an incorrect slug, "lisle" should match /^l-isle-?(bis|bahnhof|gare|stazione|ch)?$/
Station Crêt-d’y-Bau (36849) has an incorrect slug, "cret-dy-bau" should match /^cret-d-y-bau-?(bis|bahnhof|gare|stazione|ch)?$/
Station Château-d’Hauteville (36859) has an incorrect slug, "chateau-dhauteville" should match /^chateau-d-hauteville-?(bis|bahnhof|gare|stazione|ch)?$/
Station Château-d’Oex La Palaz (36887) has an incorrect slug, "chateau-doex-la-palaz" should match /^chateau-d-oex-la-palaz-?(bis|bahnhof|gare|stazione|ch)?$/
Station Val-d’Illiez (37026) has an incorrect slug, "val-dilliez" should match /^val-d-illiez-?(bis|bahnhof|gare|stazione|ch)?$/
Station La Presta Mines d’asphalte (37060) has an incorrect slug, "la-presta-mines-dasphalte" should match /^la-presta-mines-d-asphalte-?(bis|bahnhof|gare|stazione|ch)?$/
Station Vers-l’Eglise (37114) has an incorrect slug, "vers-leglise" should match /^vers-l-eglise-?(bis|bahnhof|gare|stazione|ch)?$/
Station Ondallaz-L’Alliaz (37163) has an incorrect slug, "ondallaz-lalliaz" should match /^ondallaz-l-alliaz-?(bis|bahnhof|gare|stazione|ch)?$/
Station Volochys’k (37761) has an incorrect slug, "volochysk" should match /^volochys-k-?(bis|ua)?$/
Station Sant’Agnello (37764) has an incorrect slug, "santagnello" should match /^sant-agnello-?(bis|stazione|it)?$/
Station Volyns’ka oblast (37766) has an incorrect slug, "volynska-oblast" should match /^volyns-ka-oblast-?(bis|ua)?$/
Station Sverdlovs’k (37770) has an incorrect slug, "sverdlovsk" should match /^sverdlovs-k-?(bis|ua)?$/
Station Platja d’Aro (37868) has an incorrect slug, "platja-daro" should match /^platja-d-aro-?(bis|estacion|ciudad|es)?$/
Station Bellmunt d’Urgell (37919) has an incorrect slug, "bellmunt-durgell" should match /^bellmunt-d-urgell-?(bis|estacion|ciudad|es)?$/
Station L’Aldea (38013) has an incorrect slug, "laldea" should match /^l-aldea-?(bis|estacion|ciudad|es)?$/
Station L’Alfàs del Pi-Platja (38014) has an incorrect slug, "lalfas-del-pi-platja" should match /^l-alfas-del-pi-platja-?(bis|estacion|ciudad|es)?$/
Station Saint-Cyr-l’École (38085) has an incorrect slug, "saint-cyr-lecole" should match /^saint-cyr-l-ecole-?(bis|gare|fr)?$/
Station Loix Château d’eau (39387) has an incorrect slug, "loix-chateau-deau" should match /^loix-chateau-d-eau-?(bis|gare|fr)?$/
Station Genève Musée d’art et d’histoire (47328) has an incorrect slug, "geneve-musee-dart-et-dhistoire" should match /^geneve-musee-d-art-et-d-histoire-?(bis|bahnhof|gare|stazione|ch)?$/
Station Genève Merle-d’Aubigné (47331) has an incorrect slug, "geneve-merle-daubigne" should match /^geneve-merle-d-aubigne-?(bis|bahnhof|gare|stazione|ch)?$/
Station Genève Musée d’ethnographie (47338) has an incorrect slug, "geneve-musee-dethnographie" should match /^geneve-musee-d-ethnographie-?(bis|bahnhof|gare|stazione|ch)?$/
Station Reggio all’Angelo (66415) has an incorrect slug, "reggio-allangelo" should match /^reggio-all-angelo-?(bis|stazione|it)?$/
Station Ercolano - Miglio D’Oro (66530) has an incorrect slug, "ercolano-miglio-doro" should match /^ercolano-miglio-d-oro-?(bis|stazione|it)?$/
Station Madonna Dell’Arco (66565) has an incorrect slug, "madonna-dellarco" should match /^madonna-dell-arco-?(bis|stazione|it)?$/
Station Pomigliano D’Arco (66618) has an incorrect slug, "pomigliano-darco" should match /^pomigliano-d-arco-?(bis|stazione|it)?$/
Station Ranzo-Sant’Abbondio (66640) has an incorrect slug, "ranzo-santabbondio" should match /^ranzo-sant-abbondio-?(bis|bahnhof|gare|stazione|ch)?$/
Station Sant’Anastasia (66651) has an incorrect slug, "santanastasia" should match /^sant-anastasia-?(bis|stazione|it)?$/
Station Sant’Antonino (CH) (66663) has an incorrect slug, "santantonino-ch" should match /^sant-antonino-ch-?(bis|bahnhof|gare|stazione|ch)?$/
Station Via Sant’Antonio (66703) has an incorrect slug, "via-santantonio" should match /^via-sant-antonio-?(bis|stazione|it)?$/
Station Sant’Antonino (66723) has an incorrect slug, "santantonino" should match /^sant-antonino-?(bis|stazione|it)?$/
Station Sant’Olcese Chiesa (66724) has an incorrect slug, "santolcese-chiesa" should match /^sant-olcese-chiesa-?(bis|stazione|it)?$/
Station Sant’Olcese Tullo (66725) has an incorrect slug, "santolcese-tullo" should match /^sant-olcese-tullo-?(bis|stazione|it)?$/
Station Sant’Oreste (66772) has an incorrect slug, "santoreste" should match /^sant-oreste-?(bis|stazione|it)?$/
Station Adrano - Piazza Sant’Agostino (66803) has an incorrect slug, "adrano-piazza-santagostino" should match /^adrano-piazza-sant-agostino-?(bis|stazione|it)?$/
Station Crocetta d’Orero (66827) has an incorrect slug, "crocetta-dorero" should match /^crocetta-d-orero-?(bis|stazione|it)?$/
Station Tivoli Villa d’Este (67304) has an incorrect slug, "tivoli-villa-deste" should match /^tivoli-villa-d-este-?(bis|stazione|it)?$/
Station Karl z’Neuhub Bahnhst (67564) has an incorrect slug, "karl-zneuhub-bahnhst" should match /^karl-z-neuhub-bahnhst-?(bis|at)?$/
Station Borgo d’Ale (67616) has an incorrect slug, "borgo-dale" should match /^borgo-d-ale-?(bis|stazione|it)?$/
Station Forza d’Agrò (67671) has an incorrect slug, "forza-dagro" should match /^forza-d-agro-?(bis|stazione|it)?$/
Station Sant’Alessio Siculo (67677) has an incorrect slug, "santalessio-siculo" should match /^sant-alessio-siculo-?(bis|stazione|it)?$/
Station Motta D’affermo (67682) has an incorrect slug, "motta-daffermo" should match /^motta-d-affermo-?(bis|stazione|it)?$/
Station Kotel’va (67704) has an incorrect slug, "kotelva" should match /^kotel-va-?(bis|ua)?$/
Station Tal’ne (67726) has an incorrect slug, "talne" should match /^tal-ne-?(bis|ua)?$/
Station Mel’nytsya-Podil’s’ka (67739) has an incorrect slug, "melnytsya-podilska" should match /^mel-nytsya-podil-s-ka-?(bis|ua)?$/
Station Monastyrys’ka (67740) has an incorrect slug, "monastyryska" should match /^monastyrys-ka-?(bis|ua)?$/
Station Novoarkhanhel’s’k (67741) has an incorrect slug, "novoarkhanhelsk" should match /^novoarkhanhel-s-k-?(bis|ua)?$/
Station Castell’Umberto (67767) has an incorrect slug, "castellumberto" should match /^castell-umberto-?(bis|stazione|it)?$/
Station Assisi Piazza Unità d’Italia (67842) has an incorrect slug, "assisi-piazza-unita-ditalia" should match /^assisi-piazza-unita-d-italia-?(bis|stazione|it)?$/
Station Sant Sadurní d’Anoia (74457) has an incorrect slug, "sant-sadurni-danoia" should match /^sant-sadurni-d-anoia-?(bis|estacion|ciudad|es)?$/
Station Caldes d’Estrac (74504) has an incorrect slug, "caldes-destrac" should match /^caldes-d-estrac-?(bis|estacion|ciudad|es)?$/
Station L’Ènova-Manuel (74509) has an incorrect slug, "lenova-manuel" should match /^l-enova-manuel-?(bis|estacion|ciudad|es)?$/
Station Xirivella-l’Alter (74522) has an incorrect slug, "xirivella-lalter" should match /^xirivella-l-alter-?(bis|estacion|ciudad|es)?$/
```

Reply:
We could do this but it adds 20s of runtime for the tests: I guess compiling a regex is expensive.
```diff
diff --git a/test_data.rb b/test_data.rb
index 6158fe4..fbc75b5 100644
--- a/test_data.rb
+++ b/test_data.rb
@@ -57,10 +57,14 @@ STATIONS.each do |row|
   SLUG_COUNT["#{row["slug"]}_#{row["country"]}"] += 1
 end
 
-def slugify(name)
+def old_slugify(name)
   name.gsub(/[\/\.]/,"-").to_ascii.to_url
 end
 
+def slugify(name)
+  name.gsub(/[\/\.’]/,"-").to_ascii.to_url
+end
+
 #TAP TSI: ANNEX B.9
 #STANDARD NUMERICAL CODING OF LOCATIONS
 def uic8_valid?(uic8)
@@ -382,6 +386,7 @@ class StationsTest < Minitest::Test
     end
   end
 
+  focus
   def test_correct_slugs
     suffixes = {}
     STATIONS.each do |row|
@@ -392,7 +397,8 @@ class StationsTest < Minitest::Test
         country_suffixes << country.downcase
         suffixes[country] = country_suffixes.join("|")
       end
-      assert_match(/^#{slugify(row["name"])}-?(#{suffixes[country]})?$/, row["slug"], "Station #{row["name"]} (#{row["id"]}) has an incorrect slug")
+      regex = /^(#{slugify(row["name"])}|#{old_slugify(row["name"])})-?(#{suffixes[country]})?$/
+      assert_match(regex, row["slug"], "Station #{row["name"]} (#{row["id"]}) has an incorrect slug")
     end
   end
 
```

Reply:
:thinking: yeah, let's leave things as they are.